addons/website_event_questions/views/event_question_views.xml

```diff
@@ -7,14 +7,10 @@
             <form string="Question">
                 <sheet>
                     <h1><field name="title" placeholder='e.g. "Do you have any diet restrictions?"' /></h1>
-                    <group>
-                        <group>
-                            <div colspan="2">
-                                <field name="once_per_order"/>
-                                <label for="once_per_order"/>
-                            </div>
-                            <field name="question_type" widget="radio" options="{'horizontal': true}" />
-                        </group>
+                    <group col="4">

```

Comment:
Could we instead go into a regular 

```xml
<group>
    <group>
        [fields]
    </group>
</group>
```

That way we get all fields aligned on the left?
Playing with col is not very common. 

(Same remark for the other form view)